src/bicep/modules/policies/CMMC-policyAssignmentParameters.json

```diff
@@ -5,12 +5,6 @@
     "effect-09024ccc-0c5f-475e-9457-b7c0d9ed487b" : { 
         "value" : "AuditIfNotExists"
     },
-    "MembersToExclude-69bf4abd-ca1e-4cf6-8b5a-762d42e61d4f" :{
-        "value": ""
-    },
-    "MembersToInclude-30f71ea1-ac77-4f26-9fc5-2d926bbd4ba7": {
-        "value": ""
-    },
```

Comment:
Are these changes related to tagging? If not, it'd make things easier to trace in the log in the future if they were removed and committed in a change specific to updating policy.

Reply:
Not related, it was a bug I quickly found and ran a test but not completed my testing and shouldn't be in this commit yet. Thought I stashed them.